homeassistant/helpers/event.py

```diff
@@ -1397,6 +1397,7 @@ def async_track_time_interval(
     hass: HomeAssistant,
     action: Callable[[datetime], Coroutine[Any, Any, None] | None],
     interval: timedelta,
+    *,
```

Comment:
This is the main change.
If approved, I think it should go into `2023.4` to avoid a breaking change.